lib/models/BucketInfo.ts

```diff
@@ -240,6 +243,11 @@ export default class BucketInfo {
             READ_ACP: [],
         };
 
+        if (tags === undefined) {
+            tags = [];
```

Comment:
```suggestion
            tags = [] as BucketTag[];
```

I think you need something like this to signify it is of a specific type, even if empty

Reply:
It was not needed by the transpiler but it's good to be explicit about the type https://github.com/scality/Arsenal/pull/1933/commits/4b19cc2101f4a15666f8746ad8fb12533e9b45d8#diff-4ee3cd5de98073dadb72b27f034f4979480c04388cc4f4538f4895373527f174R247t